composer.json

```diff
@@ -44,8 +43,6 @@
     },
     "scripts": {
         "test": "phpunit",
```

Comment:
I think this comment is causing an error during in the tests, can we get this removed?

Reply:
Fixed